examples/fast_style_transfer/sketch.js

```diff
@@ -0,0 +1,49 @@
+/*
+===
+Fast Style Transfer Simple Example
+===
+*/
+
+let net;
+let inputImg;
+let outputImgData;
+
+function setup() {
+  createCanvas(252, 252);
+  net = new p5ml.TransformNet(modelLoaded, 'udnie', 'models/udnie/');
+}
+
+// A function to be called when the model has been loaded
+function modelLoaded() {
+  console.log('Model loaded!');
+
+  // Set style for the model
+  net.setStyle('udnie');
+
+  inputImg = select('#input-img').elt;
+
+  /**
+  * @param inputImg HTMLImageElement of input img
+  * @return Array3D containing pixels of output img
+  */
+  outputImgData = net.predict(inputImg);
```

Comment:
It's tricky b/c I think the library should be able to be "p5 independent" and work with plain JS or other JS frameworks, however, for beginners using p5 it would be nice to be able to pass in a `p5.Image` or valid `p5.Element`/`p5.MediaElement` and receive a `p5.Image` back.

Reply:
I totally agree. I added another function called `array3DToP5Image(imageData)` which takes an Array3D and returns a p5.Image. I left this `array3DToP5Image` in the example code. I was going to include it in the p5ml library, but in order to do that, I need to add p5.js as a dependency to p5.ml. And I'm not sure if p5ml should depend on p5.js. I think this is a question that worth discussion. So I left the function in the example code for now.